covdefaults.py

```diff
@@ -58,22 +59,35 @@ def _plat_impl_pragmas():  # type: () -> List[str]
 
 
 class CovDefaults(CoveragePlugin):
-    def __init__(self, subtract_omit: str = '') -> None:
+    def __init__(
+        self,
+        subtract_omit: str = '',
+        installed_package: str = '',
+    ) -> None:
         self._subtract_omit = subtract_omit.split()
+        self._installed_package = self._parse_inst_pkg(installed_package)
+
+    @staticmethod
+    def _parse_inst_pkg(installed_package: str) -> Dict[str, Optional[str]]:
+        result: Dict[str, Optional[str]] = {}
+        for entry in installed_package.split(' '):
```

Comment:
`.split()` is better, that way you can newline separate or whitespace separate (this is in line with how coverage handles things)